doc/module-docs/cc_ansible/data.yaml

```diff
@@ -1,17 +1,16 @@
 cc_ansible:
-  name: Ansible
-  title: Configure ansible for instance
-  description:
-     This module provides ``ansible`` integration for
-     augmenting cloud-init's configuration of the local
-     node.
-
+  description: >
+    This module provides Ansible integration for augmenting cloud-init's
+    configuration of the local node.
 
-     This module installs ansible during boot and
-     then uses ``ansible-pull`` to run the playbook
-     repository at the remote URL.
+    This module installs ``ansible`` during boot and then uses ``ansible-pull``
+    to run the playbook repository at the remote URL.
   examples:
-  - comment: ""
+  - comment: |
+      Example 1:
```

Comment:
Given we are adding the `Example 1` prefix to data.yaml and data.yaml is sourced for the command line `cloud-init schema --docs all` we need to drop the example header string we define for each command line markdown example generated to avoid duplcated Example * lines
```-        rst_content += SCHEMA_EXAMPLES_SPACER_TEMPLATE.format(
-            example_count=count
-        )
```